doc/modules/mod_extdisco.md

```diff
@@ -0,0 +1,36 @@
+### Module Description
+Implements [XEP-0215: External Service Discovery](http://xmpp.org/extensions/xep-0215.html) for discovering information about services external to the XMPP network.
+The main use-case is to help discover STUN/TURN servers to allow for negotiating media exchanges.
+
+### Options
+* **type** (atom, default: unset) - service type
+
+    * **Valid values:** `stun, turn`
```

Comment:
See example 2 from https://xmpp.org/extensions/xep-0215.html#all
the last service is actually of type `ftp` 🤔